src/tests/unit/tests/DetailsView/reports/components/report-sections/footer.test.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+import { Footer } from '../../../../../../../DetailsView/reports/components/report-sections/footer';
+
+describe('Footer', () => {
+    it('renders', () => {
+        const wrapper = shallow(<Footer />);
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Should be 
```suggestion
        expect(wrapper.getElement()).toMatchSnapshot();
```
to generate a useful snapshot above?

Reply:
Good catch... looks like I'm going to fast.